tests/test_language_metadata.py

```
@@ -0,0 +1,344 @@
+
+        seen = {}
+
+        def fake_summarize(file_path, language, content, **kwargs):
+            seen["language"] = language
+            return {"file_path": file_path, "description": "ok"}
+
+        with patch(
+            "repo2readme.services.summarization.summarize_file",
+            side_effect=fake_summarize,
+        ):
+            summaries, errors = generate_all_summaries(
```

Ruff (0.16.1)

[warning] Unpacked variable `summaries` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)